File: Ex. 34.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char const *argv[])
{
    int n;
    int numeros[10];

    printf("Digite 10 numeros:\n");
    
    for(int i=0; i<10; i++)
    {
        scanf("%i", &numeros[i]);

        if (numeros[i]>n)
        {
            n=numeros[i];
        }
        
    }
    printf("\nO maior numero digitado foi: %i", n);
}

File: Python/Ex 34.py
m = 0
numeros = []
for i in range ( 0 , 10 , 1 ):
    x  =  int ( input ('Digite um numero:'))
    numeros.append(x)
 
print ('Os numeros que você digitou foram:')

for i in range ( 0 , 10 , 1 ):
    if ( numeros [ i ] > m ):
        m = numeros [ i ]

print('o maior numero digitado foi:')
print(f'{m}; ')